scripts/auth.yml.example

```diff
@@ -1,2 +1,5 @@
 google:
     key: YOUR_KEY
+pypi:
```

Comment:
It looks like it does work with pypirc, fyi:

https://github.com/pypa/twine/blob/master/twine/utils.py